### PR TITLE
chore: add missing props

### PR DESCRIPTION
**Pull request by ravinderk**
### Description of the Change
- Add missing props in `CHANGELOG.md` and `readme.txt`
- Add Github compare link to release `1.5.3` in `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ### Fixed
 
-- PHP 8.2 deprecation warnings
+- PHP 8.2 deprecation warnings (props [@dsawardekar](https://github.com/dsawardekar))
```
Comment (peterwilsoncc) [inline CHANGELOG.md]: Full stop for consistency. This applies to the rest of this file too.

```suggestion
- PHP 8.2 deprecation warnings (props [@dsawardekar](https://github.com/dsawardekar)).
```

File: readme.txt [hunk 2]
```diff
@@ -56,9 +56,9 @@ To address this you need to update your custom block registration. If this is ou
 
 = 1.5.3 - 2023-11-23 =
 
-* **Fixed:** PHP 8.2 deprecation warnings
-* **Added:** PHPUnit 9.x support
-* **Security:** Bumps sharp from 0.32.3 to 0.32.6
+* **Fixed:** PHP 8.2 deprecation warnings (props [@dsawardekar](https://github.com/dsawardekar))
+* **Added:** PHPUnit 9.x support (props [@dsawardekar](https://github.com/dsawardekar))
+* **Security:** Bumps sharp from 0.32.3 to 0.32.6 (props [@dependabot](https://github.com/apps/dependabot))
```
Comment (peterwilsoncc) [inline readme.txt]: As above

```suggestion
* **Fixed:** PHP 8.2 deprecation warnings (props [@dsawardekar](https://github.com/dsawardekar)).
* **Added:** PHPUnit 9.x support (props [@dsawardekar](https://github.com/dsawardekar)).
* **Security:** Bumps sharp from 0.32.3 to 0.32.6 (props [@dependabot](https://github.com/apps/dependabot)).
```

**Review by peterwilsoncc — COMMENTED** [review]
Style nitpicks, otherwise looks goo.

**Comment by peterwilsoncc** [comment]
The same changes are needed elsewhere in changelog.md

**Comment by ravinderk** [comment]
> The same changes are needed elsewhere in changelog.md

@peterwilsoncc fixed: https://github.com/10up/block-catalog/pull/37/commits/c4d44621f1d665a63f3887e9aa5d4f64d08bc34d

**Review by peterwilsoncc — APPROVED** [review]

**Pull request merged**